tidb-server/main.go

```diff
@@ -331,9 +360,6 @@ func reloadConfig(nc, c *config.Config) {
 	// like config.GetGlobalConfig().OOMAction.
 	// These config items will become available naturally after the global config pointer
 	// is updated in function ReloadGlobalConfig.
-	if nc.Performance.MaxProcs != c.Performance.MaxProcs {
```

Comment:
Can we check whether we've set CPUAffinity, if so, we do not reset max-procs?

Reply:
After this change, config `MaxProcs` in the config has no effect? @lysu

Reply:
@XuHuaiyu  you mean using `taskset` + `-affinity-cpus` at the same time?  taskset is too hard to use, normally only manually set in some test env, after this pr will advise people using `-affinity-cpus` by issue a follow up tidb-ansible pr.

Reply:
@tiancaiamao this PR just want to reduce working to "set MaxProcs" when binding CPU cores, if a user doesn't bind cores can keep using MaxProcs as its like, but when a user bind cores, in 99% they want to use binded core.